src/app/core/admin/admin-entity-details/admin-entity-form/admin-entity-form.component.spec.ts

```diff
@@ -180,4 +180,12 @@ describe("AdminEntityFormComponent", () => {
       jasmine.arrayContaining(removedFields),
     );
   }));
+
+  fit("should hide a single field", fakeAsync(() => {
```

Comment:
```suggestion
  it("should hide a single field", fakeAsync(() => {
```

don't forget to remove the "focus" from tests at the end, so that all tests get run